File: C++/program118.cpp
#include<iostream>
using namespace std;
int main()
{
    int Arr[5] = { '\0'};
    int iCnt = 0;

    cout<<"Enter array elements "<<'\n';
    for(iCnt = 0; iCnt < 5; iCnt++)
    {
        cin>>Arr[iCnt];
    }
    
    

    cout<<"Elements from array are "<<'\n';
    for(iCnt = 0; iCnt < 5; iCnt++)
    {
        cout<<Arr[iCnt] <<'\n';
    }
    
    return 0;
}

File: Java/program118.java
import java.util.*;

class program118
{
    public static void main(String A[])
    {
       
        int [] Arr = new int [5];
        int iCnt = 0;
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter elements :");

        for(iCnt = 0; iCnt < 5 ; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt() ;
        }
       

        System.out.println("Elements from array are :");
        
        for(iCnt = 0; iCnt < 5 ; iCnt++)
        {
            System.out.println(Arr[iCnt]);
        }
    }
}